pkg/virt-controller/watch/application.go

```diff
@@ -311,10 +322,12 @@ func Execute() {
 	app.kubeVirtInformer = app.informerFactory.KubeVirt()
 	app.informerFactory.Start(stopChan)
 
-	app.kubeVirtInformer.SetWatchErrorHandler(func(r *cache.Reflector, err error) {
+	if err := app.kubeVirtInformer.SetWatchErrorHandler(func(r *cache.Reflector, err error) {
```

Comment:
I think this will always return an error when this method is called on an informer which has already started:
https://github.com/kubevirt/kubevirt/blob/692d3801a89413a417e1d7553e469f4010b6779a/vendor/k8s.io/client-go/tools/cache/shared_informer.go#L356-L366

Can you move this line before:
```go
app.informerFactory.Start(stopChan)
```

Then It should work as expected.

Reply:
Amazing catch @akrejcir , thanks.

Done.